src/utils.jl

```diff
@@ -417,13 +417,6 @@ function combinearg(arg_name, arg_type, is_splat, default)
     return default === nothing ? a2 : Expr(:kw, a2, default)
 end
 
-
-macro splitcombine(fundef)
```

Comment:
The macro with the same name is defined in test file, and this
could be removed.
This definition is also timed, especially `rebuilddef` is no longer defined.